chapters/bash-advanced.Rmd

```diff
@@ -1047,13 +1046,9 @@ while @Jans2014 looks specifically at how to process data on the command line.
 
 ## Exercises {#bash-advanced-exercises}
 
-Many of the exercises below have been adapted from Software Carpentry's lesson
-[The Unix Shell][swc-shell-novice].
```

Comment:
Do we mention this elsehwere?

Reply:
In the Acknowledgements section of the Welcome chapter we say: "many of the explanations and exercises in Chapters 2–4 have been adapted from Software Carpentry’s lesson The Unix Shell."